### PR TITLE
add MYB space

### PR DESCRIPTION
**Pull request by cryptnotiq**
https://mybit.io/
https://twitter.com/MyBit_DApp
https://t.me/mybitio

### CODE REVIEW

File: spaces/mybit/index.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+  "key": "mybit",
+  "symbol": "MYB",
+  "name": "MyBit",
+  "defaultView": "core",
+  "address": "0x5d60d8d7ef6d37e16ebabc324de3be57f135e0bc",
```
Comment (bonustrack) [inline spaces/mybit/index.json:6]: Something is wrong with the token, the call to query `decimals()` is failing. Is this a standard ERC20 token?

Comment (dev-xu) [reply spaces/mybit/index.json:6]: @bonustrack Thanks. I am Roy from Mybit. We don't have `decimals()` method in our token as it is optional in ERC-20.

Comment (bonustrack) [reply spaces/mybit/index.json:6]: You'll need to add your token here: https://github.com/balancer-labs/snapshot/blob/develop/src/store/modules/web3.ts#L310 and in line 327
This is quickfix, later we will stop request decimals from space token.

Comment (dev-xu) [reply spaces/mybit/index.json:6]: @bonustrack Made a PR for what you suggested. Thanks

**Review by e-asia — APPROVED** [review]

**Pull request merged**